### PR TITLE
Saved places

### PR DESCRIPTION
**Pull request by emmapaczkowski**
Add a functionality to save a trip to the database ans display a trip.  There is still a lot of integration with the UI to get this working.

### CODE REVIEW

File: where-to-travel/src/main/webapp/script.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ let globalNonce;
 // Keep a set of all saved and displayed places
 let savedPlacesSet = new Set();
 let displayedPlacesSet = new Set();
+let triptsSet = new Set();
```
Comment (PriyaVarra) [inline where-to-travel/src/main/webapp/script.js]: triptsSet -> tripsSet

File: where-to-travel/src/main/webapp/script.js [hunk 2]
```diff
@@ -1332,4 +1334,61 @@ function removeMorePlaceInfo(place_id) {
       `<a onclick="populateMorePlaceInfo('${place_id}')" class="btn btn-primary active">
          More Information
        </a>`;
-}
\ No newline at end of file
+}
+
+//Function that needs to be called when the saved places togle is turned on.
+//TODO:call this function
+function querrySavedTrips() {
```
Comment (PriyaVarra) [inline where-to-travel/src/main/webapp/script.js]: nit: querry -> query

File: where-to-travel/src/main/webapp/script.js (hunk 1 above)
Comment (emmapaczkowski) [reply where-to-travel/src/main/webapp/script.js]: done

File: where-to-travel/src/main/webapp/script.js (hunk 2 above)
Comment (emmapaczkowski) [reply where-to-travel/src/main/webapp/script.js]: done

**Pull request merged**